.travis.yml

```
@@ -40,4 +40,4 @@ jobs:
         on:
           branch: master
         script:
-          - npx semantic-release
+          - yarn && yarn build && npx semantic-release
```

```suggestion
          - yarn
          - yarn build
          - npx semantic-release
```
Not sure on this one, but `stage: build` does it this way :)

---

As it was pointed to me, https://github.com/travis-ci/travis-ci/issues/1066. My bad 🤗